src/Postcode.php

```diff
@@ -171,7 +171,7 @@ public function setCountry($country = null)
     public function mutatePostcode($postcode)
     {
         // Ensure the postcode is all upper case with no leading or trailing spaces
-        return strtoupper(trim($postcode));
+        return strtoupper(preg_replace('/[ \n\t]+/i', '', trim($postcode)));
```

Comment:
why was this change necessary? This will remove the white spaces within the postcode as well which could cause side effects

Reply:
I know It replaces the white space... your test units would fail without it, If your test units are not up-to-date I can revert this change.

Reply:
okay it looks good